<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>new_user_window</class>
 <widget class="QWidget" name="new_user_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>186</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Создание нового пользователя</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="new_user_message">
           <property name="text">
            <string>Введите имя</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="psw1_message">
           <property name="text">
            <string>Введите пароль</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="psw2_message">
           <property name="text">
            <string>Повторите пароль</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLineEdit" name="new_user_name"/>
         </item>
         <item>
          <widget class="QLineEdit" name="psw1_input">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="psw2_input">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="error_message">
       <property name="text">
        <string/>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
